start.py

```diff
@@ -15,6 +15,7 @@
 import os
 import textwrap
 
+from packaging.version import Version
```

Comment:
Things in the standard library should be in the first block of imports (https://pep8.org/#imports)